Validator/IsFqdn.cs

```diff
@@ -13,52 +13,43 @@ public partial class Validator
         /// <param name="allowUnderscore"></param>
         /// <param name="allowTrailingDot"></param>
         /// <returns></returns>
-        public static bool IsFqdn(
-            string input,
-            bool requireTld = true,
-            bool allowUnderscore = false,
-            bool allowTrailingDot = false)
+        public static bool IsFqdn(string input, bool requireTld = true, bool allowUnderscore = false, bool allowTrailingDot = false)
```

Comment:
@mgmccarthy This refactor runs afoul of the project [guideline](https://github.com/AlexArchive/Validator/blob/master/Contributing.Md#code-guidelines) of keeping each line under 80 characters wide, please update it so that it is shorter than 80 characters